### ISSUE TITLE
Can't get zone spawn to work (1.7.10 / FE 1.4.6.1348)

### ISSUE THREAD

**Issue by jonnybjonnyb** [ForgeEssentials/ForgeEssentials#2335]
Trying to implement zone spawn (if you die in Zone X, spawn at Location Y -- via /p global zone  ZoneX spawn here). On my server, when a player dies in Zone X, he respawns at his bed rather than Location Y. Guessing/wondering that the bed spawn permissions (or some other global perms?) are higher priority than the zone permissions? 

The zone spawn location seems correct in permissions.json. Can't figure out if permission priorities are causing the issue or if it's something else. Permissions.json here: https://pastebin.com/UdExBXyG - zone spawn attempt (zone = "gauntlet") highlighted. I'm aware this certainly could be user error.

**Comment by LoganDark** [comment]
Beds always take priority over zone spawns, I think.

**Comment by jonnybjonnyb** [comment]
Yeah, that would explain it. Maybe I'll try disabling bed spawn point and see if manual setting of home/spawn also takes priority over zone spawn.

Thanks.

**Comment by Zethalion** [comment]
Is this still an issue on 1.12.2 ? Closing this issue in 7 days if no respone @jonnybjonnyb
